.github/workflows/check_enforcer.yml

```diff
@@ -12,7 +12,8 @@ jobs:
       contents: read
     steps:
       - uses: actions/checkout@v4
-      - run: git fetch origin main
+        with:
```

Comment:
how will we test this change?

Reply:
This branch is out of date. but check enforcer is passed.